clang/lib/3C/ConstraintVariables.cpp

```diff
@@ -78,9 +78,10 @@ PointerVariableConstraint::derefPVConstraint(PointerVariableConstraint *PVC) {
   assert(!PVC->Vars.empty() && !SrcVars.empty());
   Vars.erase(Vars.begin());
   SrcVars.erase(SrcVars.begin());
-  return new PointerVariableConstraint(Vars, SrcVars, PVC->getTy(),
-                                       PVC->getName(), PVC->getFV(),
-                                       PVC->getItype());
+  return new PointerVariableConstraint(
```

Comment:
Since you do this twice, (Also on line 104), do you think it's worth adding a constructor to PVC that just takes a PVC and pulls the fields out itself? If we're not likely to add more calls like this it might not be.